src/app/app.component.html

```diff
@@ -1 +1 @@
-<stack-details [stack]="codebases && codebases[0]"></stack-details>
\ No newline at end of file
+<stack-details [stack]="stackUrl && stackUrl"></stack-details>
```

Comment:
Yes. It is just an extra check. I will handle the EOL. I think that check can be there.